src/effects/something_rotating.c

```diff
@@ -1,6 +1,66 @@
 #include "common.h"
 #include "effects_internal.h"
 
+s32 D_E0116C60[] = { 0x09004458 };
```

Comment:
These should be made into symbols and added to undefined syms instead of being raw pointers (same as the ones below), and the arrays should be typed as Gfx*[]